src/react/users.tsx

```diff
@@ -1,17 +1,23 @@
 import axios from 'axios';
 import { useEffect, useState } from 'react';
-import avatar from "../images/avatar.jpg";
+const avatar = require ("../images/avatar.jpg");
 
-function UsersList() {
-  const [users, setUsers] = useState([])
-  const [error, setError] =useState(null)
+interface username {
+  idUser: number;
+  FirstName: string;
+  LastName: string;
+}
```

Comment:
This interface has Users properties. Username - stands for name of the user. Consider renaming the interface to User